docs/Using-Fleet/Permissions.md

```diff
@@ -12,45 +12,43 @@ Users with the Admin role receive all permissions.
-| Filter hosts using policies                          | ✅       | ✅         | ✅    |
 | Target hosts using labels                            | ✅       | ✅         | ✅    |
-| Run saved queries as live queries against all hosts  | ✅       | ✅         | ✅    |
-| Run custom queries as live queries against all hosts |          | ✅         | ✅    |
 | Enroll hosts                                         |          | ✅         | ✅    |
 | Delete hosts                                         |          | ✅         | ✅    |
 | Transfer hosts between teams\*                       |          | ✅         | ✅    |
-| Create saved queries                                 |          | ✅         | ✅    |
-| Edit saved queries                                   |          | ✅         | ✅    |
-| Delete saved queries                                 |          | ✅         | ✅    |
-| Schedule queries for all hosts                       |          | ✅         | ✅    |
-| Schedule queries for all hosts assigned to a team\*  |          | ✅         | ✅    |
-| Create packs                                         |          | ✅         | ✅    |
-| Edit packs                                           |          | ✅         | ✅    |
-| Delete packs                                         |          | ✅         | ✅    |
-| Create labels                                        |          | ✅         | ✅    |
-| Edit labels                                          |          | ✅         | ✅    |
-| Delete labels                                        |          | ✅         | ✅    |
-| Add policies for all hosts                           |          | ✅         | ✅    |
-| Remove policies for all hosts                        |          | ✅         | ✅    |
-| Add policies for hosts assigned to team\*            |          | ✅         | ✅    |
-| Remove policies for hosts assigned to team\*         |          | ✅         | ✅    |
-| Create users                                         |          |            | ✅    |
-| Edit users                                           |          |            | ✅    |
-| Delete users                                         |          |            | ✅    |
-| Edit own user information                            | ✅       | ✅         | ✅    |
+| Create, edit, and Delete labels                      |          | ✅         | ✅    |
+| Browse all software                                  | ✅       | ✅         | ✅    |
+| Filter software by vulnerabilities                   | ✅       | ✅         | ✅    |
+| Filter hosts by software                             | ✅       | ✅         | ✅    |
+| Filter software by team*                             | ✅       | ✅         | ✅    |
+| Manage software by vulnerability automations         |          | ✅         | ✅    |
+| Run saved queries as live queries against all hosts  | ✅       | ✅         | ✅    |
+| Run custom queries as live queries against all hosts |          | ✅         | ✅    |
```

Comment:
```suggestion
| Run any query as live query against all hosts |          | ✅         | ✅    |
```
I suggest updating this to "any query" so that this line is clearly distinguished from the above "observer can run" line.